app/js/ui.js

```diff
@@ -1,29 +1,64 @@
 import 'whatwg-fetch';
 import React from 'react';
+import { autobind } from 'core-decorators';
+import classNames from 'classnames';
+import { App } from 'spak';
 
-class App extends React.Component {
+class AppVC extends React.Component {
     constructor() {
         super();
         this.state = {
-            styles: ''
+            styles: '',
+            hasLoginForm: false
```

Comment:
I normally don't have trailing commas just because back in the day that was an error in IE. I'm open to trying it out for this project.